internal/utils/diff/diff.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package link
+package diff
```

Comment:
Couldn't put this under `utils` because import in `config` would cause cyclic dependencies.